Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.8
+ENV PYTHONUNBUFFERED 1
+RUN mkdir /vulnerablecode
+WORKDIR /vulnerablecode
+ADD . /vulnerablecode/
+RUN pip install -r requirements.txt
```

Comment:
Can we add some labels to the image such that we know its base image, origin and license, author, maintainers, etc? (See how complicated a Docker image analysis can be with https://github.com/nexB/scancode.io/blob/main/scanpipe/pipelines/docker.py )